hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/nativeio/NativeIO.java

```diff
@@ -219,8 +219,8 @@ public static boolean isPmem(long address, long length) {
       }
 
       // create a pmem file and memory map it
-      public static PmemMappedRegion mapBlock(String path, long length) {
-        return NativeIO.POSIX.pmemCreateMapFile(path, length);
+      public static PmemMappedRegion mapBlock(String path, long length, boolean isFileExists) {
```

Comment:
Update java doc, e.g., map a file in persistent memory, if the given file exists, directly map it. If not, create the named file on persistent memory and then map it.